<!DOCTYPE html>
<html lang="en">
  <head>
    <title> Teodorescu_Lab &mdash; Bioelectronics   </title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300i,400,700" rel="stylesheet">
    <link rel="stylesheet" href="fonts/icomoon/style.css">

    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/magnific-popup.css">
    <link rel="stylesheet" href="css/jquery-ui.css">
    <link rel="stylesheet" href="css/owl.theme.default.min.css">

    <link rel="stylesheet" href="css/lightgallery.min.css">    
    
    <link rel="stylesheet" href="css/bootstrap-datepicker.css">

    <link rel="stylesheet" href="fonts/flaticon/font/flaticon.css">
    
    <link rel="stylesheet" href="css/swiper.css">

    <link rel="stylesheet" href="css/aos.css">

    <link rel="stylesheet" href="css/style.css">
    
  </head>
  <body>
  
  <div class="site-wrap">

    <div class="site-mobile-menu">
      <div class="site-mobile-menu-header">
        <div class="site-mobile-menu-close mt-3">
          <span class="icon-close2 js-menu-toggle"></span>
        </div>
      </div>
      <div class="site-mobile-menu-body"></div>
    </div>
    



    <header class="site-navbar py-3" role="banner">

      <div class="container-fluid">
        <div class="row align-items-center">
          
          <div class="col-6 col-xl-2" data-aos="fade-down">
              <h3 style="color:#396791";>Teodorescu Lab </h3>
<!--            <h1 class="mb-0"><a href="index.html"><img src="images/logo.png" width="250" title="Danser Lab Logo" alt="Danser"></a></h1> -->
            <h1 class="mb-0"><a href="index.html"><img src="images/UCSC_logo.jpg" width="200" title="UCSC Logo" alt="UCSC Logo"></a></h1>

          </div>
          <div class="col-10 col-md-8 d-none d-xl-block" data-aos="fade-down">
            <nav class="site-navigation position-relative text-right text-lg-center" role="navigation">

              <ul class="site-menu js-clone-nav mx-auto d-none d-lg-block">
                             <li class="active"><a href="index.html">Home</a></li>
                             <li class="has-children">
                               <a>Research</a>
                                                <ul class="dropdown">
                                                  <li><a href="https://braingeneers.ucsc.edu/">Braingeneers</a></li>
                                                  <li><a href="Bioelectronics.html">Bioelectronics</a></li>
                                                  <li class="has-children">
                                                    <a href="DANSER.html">DANSER Lab</a>
                                                    <ul class="dropdown">
                                                          <li><a href="Robotics.html">Robotics</a></li>
                                                          <li><a href="VR.html">Virtual and Augmented Reality</a></li>
                                                          <li><a href="Earth.html">Earth Science </a></li>
                                                       <li class="has-children">
<!--                                                       <a href="#">Aeronautics</a>
                                                           <ul class="dropdown">
                                                           <li><a href="Aerolastic.html">Aerolastic Control</a></li>
                                                           <li><a href="Guidance.html">Guidance and Navigation</a></li>
                                                           </ul>
-->
                                                    </ul>
                                                  </li>
                                                </ul>
                                              </li>
                                              <li><a href="labmembers.html">Lab Members</a></li>
                                              <li><a href="publications.html">Publications</a></li>
                                              <li><a href="https://www.youtube.com/channel/UCAqXG_Iffs5zALaOvm6T2Rg">Videos</a></li>
                                              <li><a href="https://www.soe.ucsc.edu/people/mteodorescu">Contact</a></li>
                                              <li><a href="Openings.html">Current openings</a></li>

                                            </ul>
                                          </nav>
                                        </div>

            <div class="d-inline-block d-xl-none ml-md-0 mr-auto py-3" style="position: relative; top: 3px;"><a href="#" class="site-menu-toggle js-menu-toggle text-black"><span class="icon-menu h3"></span></a></div>

          </div>

        </div>
      </div>
      
    </header>

      <div class="site-section"  data-aos="fade">
        <div class="container-fluid">
          
          <div class="row justify-content-center">
            <div class="col-md-7">
              <div class="row mb-5">
                <div class="col-12 ">
                </div>
              </div>

        
          
<! Table >
<! ---------------------------------------------------- >
            <table>
                <thead>
                    <tr>
                    <h2 class="site-section-heading text-center">Braingeneers</h2>
                    </tr>
                </thead>

              </table>
<! Ennd Table >
<! ---------- >

  <!-- <div align="center"><h3>Openings</h3></div>
       <div align="left">
       <ul>
         <p></p><li><u>PostDoctoral Research Fellows:</u><br> We have several openings on topics involving:<br> - Optics, Lensless imaging, Microimaging <br> - Bioelectronics, Microfluidics, Biomedical Integration <br> - Tissue regeneration. <br>If you are interested, please contact me and include "PostDoctoral" in the e-mail subject.</li><p></p>
         <p></p><li>
             <a href="https://recruit.ucsc.edu/JPF00815"><u>Electrical and Computer Engineering:</u></a> Junior Specialist</li><p></p>
       </ul>
       </div>
  -->
  <p><div align="center"> Please check our <a href="https://braingeneers.ucsc.edu/"><img src="images/braingeneers-logotype.png" width="150" title="Braingeneers" alt="Braingeneers" style="vertical-align:top"></a> website for additional information </div></p>
  
  
        <hr style="1px solid black;" />

  <p> <div align="center"> If you would like to join us, please check out our <a href="Openings.html">CURRENT OPENINGS</a> </div></p>
        
        <hr style="1px solid black;" />

 
 <div align="center"><h3>Research</h3></div>
 <!- Paper ->
  <!- --------------- ->
  <table>
    <thead>
        <tr>
        </p>Seiler S.T., Mantalas G.L., Selberg J., Cordero S., Torrse-Montoya S., Baudin P.V., Ly V.T., Amend F., Tran L., Hoffman R.N., Rolandi M., Green R.E, Haussler D., Salama S.R and Teodorescu M. "Modular automated microfluidic cell culture platform reduces glycolytic stress in cerebral cortex organoids" <a href="https://www.biorxiv.org/content/10.1101/2022.07.13.499938v1.abstract">bioRxiv</a>
        <p>
        </tr>
    </thead><tbody><tr style="vertical-align:top">
        <small>Abstract — Organ-on-a-chip systems combine microfluidics, cell biology, and tissue engineering to culture 3D organ-specific in vitro models that recapitulate the biology and physiology of their in vivo counterparts. Here, we have developed a multiplex platform that automates the culture of individual organoids in isolated microenvironments at user-defined media flow rates. Programmable workflows allow the use of multiple reagent reservoirs that may be applied to direct differentiation, study temporal variables, and grow cultures long term. Novel techniques in polydimethylsiloxane (PDMS) chip fabrication are described here that enable features on the upper and lower planes of a single PDMS substrate. RNA sequencing (RNA-seq) analysis of automated cerebral cortex organoid cultures shows benefits in reducing glycolytic and endoplasmic reticulum stress compared to conventional in vitro cell cultures.
    </small>
    </table>
  <p><img src="images/Spencer_1.jpg" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
      <small> <em> Design and implementation of the automated, microfluidic culture platform. (A) Illustration of the automated, microfluidic organoid culture platform, the Autoculture. (B) Front view images of the Autoculture. (1) Refrigerator with reagent reservoirs. (2) Syringe pump, distribution valves, and control interface. (3) Refrigerator with conditioned media collection reservoirs. These components reside on a lab bench directly above the cell culture incubator. (4) Microfluidic tubes enter through an incubator port and connect to the (5) microfluidic well plate chip inside the incubator. (6) Cross-sectional diagram of a single well containing an organoid culture.
      </em></small>
      <p><img src="images/Spencer_2.jpg" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
          <small> <em> Longitudinal monitoring of organoid development. (A) The Autoculture microfluidic chip sits on a remote-controlled, IoT-enabled, 24-well automated imaging system. (B) Bright-field images of twelve individual 12-day-old cerebral cortex cultures at day 1 of automated feeding. (C) Longitudinal imaging of “Culture 4” during the experiment. (D) Projected area expansion of “Culture 4” during the experiment. This was obtained using a computer vision algorithm.
          </em></small>

<hr style="1px solid black;" />



<!- Paper ->
      <!- --------------- ->
      <table>
        <thead>
            <tr>
            </p>Parks D.,  Voitiuk K.,  Geng J.,  Elliott M.A.T., Keefe M.G., Jung E.A., Robbins A., Baudin P.V., Ly V.T., Hawthorne N., Yong D., Sanso S.E., Rezaee N., Sevetson J., Seiler S.T.,  Currie R., Hengen K.B., Nowakowski T.J.,  Salama S.R., Teodorescu M., Haussler D. "Internet of Things Architecture for High Throughput Biology"  <a href="https://www.biorxiv.org/content/10.1101/2021.07.29.453595v1.abstract">bioRxiv</a>
            <p>
            </tr>
        </thead><tbody><tr style="vertical-align:top">
            <small>Abstract — New cell culture techniques have led to complex tissue models in biological experiments. For example, 3-D cerebral organoids provide a more realistic model of the human cortical tissue. However, these cell culture experiments are restricted by high costs and limited labor. A massively scalable and cost efficient platform for tissue experiments would benefit genomics, neuroscience, and translational medicine by enabling advanced high throughput tissue screens. Cloud computing and the Internet of Things (IoT) provide new tools for managing multiple experiments in parallel that are remotely controlled through automation. We introduce a cloud-based IoT architecture that takes advantage of these tools to offer an environment where researchers can run thousands of cell culture experiments at once. This technology allows studies with cell cultures to be performed at scales far beyond a single lab setting, democratizing access to advanced tissue models and enabling new avenues of research.
        </small>
        </table>
      <img src="images/Paper_DavidP.png" width="300" title="slug" alt="slug" style="vertical-align:top">

   <hr style="1px solid black;" />
   
   <!- Paper ->
    <!- --------------- ->
    <table>
      <thead>
          <tr>
          </p>Voituik K., Geng J., Keefe M.G., Parks D.F., Sanso S.E., Hawthorne N., Freeman D.B., Nowakowski T.J., Salama S.R., Teodorescu M., Haussler D. "Light-weight Electrophysiology Hardware and Software Platform for Cloud-Based Neural Recording Experiments." <a href="https://www.biorxiv.org/content/10.1101/2021.05.18.444685v1">bioRxiv</a>
          <p>
          </tr>
      </thead><tbody><tr style="vertical-align:top">
          <small>Abstract — Neural activity represents a functional readout of neurons that is increasingly important to monitor in a wide range of experiments. Extracellular recordings have emerged as a powerful technique for measuring neural activity because these methods do not lead to the destruction or degradation of the cells being measured. Current approaches to electrophysiology have a low throughput of experiments due to manual supervision and expensive equipment. This bottleneck limits broader inferences that can be achieved with numerous long-term recorded samples. \textit{Approach.} We developed Piphys, an inexpensive open source neurophysiological recording platform that consists of both hardware and software. It is easily accessed and controlled via a standard web interface through Internet of Things (IoT) protocols. \textit{Main Results.} We used a Raspberry Pi as the primary processing device \suggFinal{and}{along with an} Intan bioamplifier. We designed a hardware expansion circuit board and software to enable voltage sampling and user interaction. This standalone system was validated with primary human neurons, showing reliability in collecting \suggFinal{real-time}{} neural activity \suggFinal{}{in near real-time}.  \textit{Significance.} The hardware modules and cloud software allow for remote control of neural recording experiments as well as horizontal scalability, enabling long-term observations of development, organization, and neural activity at scale.
              
      </small>
      </table>
    <p><img src="images/Kate_Sury_1.jpg" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
        <small> <em> Piphys hardware components. (a) Expansion shield (blue board) attached on top of Raspberry Pi (green board). (b) Logic level connection. (c) Example interface with standard 6-well electrode plate. (1) +5 V logic, (2) +3.3 V logic, (3) +3.5 V logic, (4) external supply inputs, (5) Raspberry Pi input/output pins (bottom), (6) LVDS converter, (7) Intan RHD adapter, (8) Intan RHD 32 channel recording headstage containing Intan RHD2132 bioamplifier chip, (9) optional adapter board to electrodes, (10) multiple electrode types possible, (11) Raspberry Pi computer (bottom).
        </em></small>
    <p><img src="images/Kate_Sury_2.jpg" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
        <small> <em> Software overview. The software that runs on the local Raspberry Pi device communicates with the Intan RHD2132 bioamplifier chip to stream and store the digitized neural signal. Concurrently, it pushes the signal to Redis for near real-time visualization on the online dashboard. Datasets are also uploaded to S3 after each recording for permanent storage and access. Experimental control such as 'start', 'stop', and variable configuration is sent from the dashboard through Amazon IoT to the local device. Past experiment data can also be browsed using records from S3.
        </em></small>

 <hr style="1px solid black;" />

 <!- Paper ->
  <!- --------------- ->
  <table>
    <thead>
        <tr>
        </p>Ly V.T, Baudin P.V., Pansodtee P, Jung E.A., Voitiuk K, Rosen Y, Willsey H.R., Mantalas G.L., Seiler S.T., Selberg J.A., Cordero S.A., Ross J.M., Rolandi M., Pollen A.A., Nowakowski T.J, Haussler D., Mostajo-Radji M.A., Salama S.R., Teodorescu M. "Development of a Low-Cost System for Simultaneous Longitudinal Biological Imaging." <a href="https://www.biorxiv.org/content/10.1101/2021.05.17.443454v2.abstract">bioRxiv</a>
        <p>
        </tr>
    </thead><tbody><tr style="vertical-align:top">
        <small>Abstract — Simultaneous longitudinal imaging across multiple conditions and replicates has been crucial for scientific studies aiming to understand biological processes and disease. Yet, imaging systems capable of accomplishing these tasks are economically unattainable for most academic and teaching laboratories around the world. Here, we propose the Picroscope, which is the first low-cost system for simultaneous longitudinal biological imaging made primarily using off-the-shelf and 3D-printed materials. The Picroscope is compatible with standard 24-well cell culture plates and captures 3D z-stack image data. The Picroscope can be controlled remotely, allowing for automatic imaging with minimal intervention from the investigator. Here, we use this system in a range of applications. We gathered longitudinal whole organism image data for frogs, zebrafish, and planaria worms. We also gathered image data inside an incubator to observe 2D monolayers and 3D mammalian tissue culture models. Using this tool, we can measure the behavior of entire organisms or individual cells over long-time periods.
            
    </small>
    </table>
  <p><img src="images/Victoria_Pierre_2.png" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
      <small> <em> The Control Console passes commands and experiment parameters to Picroscope, which uploads results to a server allowing them to be viewed through the Image Viewer website.
      </em></small>
 <p><img src="images/Victoria_Pierre_3.png" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
      <small> <em> (a) The Picroscope inside a standard tissue culture incubator. (b) Imaging of human embryonic stem cells as a model of 2D-monolayer cell cultures. (c) Longitudinal imaging of human cortical organoids embedded in Matrigel. Zoomed images show cellular outgrowths originating in the organoids. (d) Tracking of cortical organoid development over 86 h. Images were taken hourly. On left. Images of the tracked organoid at timepoints 0, 43, and 86. On right. Measurement of organoid area at each timepoint analyzed. (e) Manual Longitudinal tracking of individual cells in embedded cortical organoids over 40 min. Images were taken every 10 min. Magenta = example of cell division, Red = example of cell migration, and Purple = example of morphological changes.
      </em></small>

<hr style="1px solid black;" />

<!- Paper ->
 <!- --------------- ->
 <table>
   <thead>
       <tr>
       </p>Baudin P.V, Ly V.T, Pansodtee P, Jung E.A, Currie R, Hoffman R, Willsey H.R, Pollen A.A, Nowakowski T.J, Haussler D., Mostajo-Radji M.A., Salama S.R., Teodorescu M. "Low cost cloud based remote microscopy for biological sciences." Elsevier Internet of Things (Special Issue: IoT Toward COVID-19) (2021) <a href="https://arxiv.org/abs/2106.07419v1">arXiv preprint arXiv:2106.07419.</a>
       <p>
       </tr>
   </thead><tbody><tr style="vertical-align:top">
       <small>Abstract — A low cost remote imaging platform for biological applications was developed. The “Picroscope” is a device that allows the user to perform longitudinal imaging studies on multi-well cell culture plates. Here we present the network architecture and software used to facilitate communication between modules within the device as well as system communication with external cloud services. A web based console was created to control the device and view experiment results. Post processing tools were developed to analyze captured data in the cloud. The result is a platform for monitoring biological experiments from outside the lab.
   </small>
   </table>
 <p><img src="images/Pierre_Victoria_1.jpg" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
     <small> <em> Graphical abstract.
     </em></small>
<p><img src="images/Pierre_Victoria_2.jpg" width="400" title="slug" alt="slug" style="vertical-align:top"></p>
         <small> <em> Human Cortical Organoid with Outgrowths: Organoids were plated on laminin coated 24 well plates and allowed to adhere to the surface. Outgrowths and cellular migration were restricted to the 2D plane of the plate’s surface and captured/monitored over the course of 21 days. Significant morphological changes occurred within the first 48 h after imaging began.
         </em></small>

<hr style="1px solid black;" />


   
                
                 
                
            </div>
          </div>
        </div>
      </div>
  
    <div class="footer py-4">
      <div class="container-fluid text-center">

      </div>
    </div>
  
      
  
      
      
    </div>
  
    <script src="js/jquery-3.3.1.min.js"></script>
    <script src="js/jquery-migrate-3.0.1.min.js"></script>
    <script src="js/jquery-ui.js"></script>
    <script src="js/popper.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/owl.carousel.min.js"></script>
    <script src="js/jquery.stellar.min.js"></script>
    <script src="js/jquery.countdown.min.js"></script>
    <script src="js/jquery.magnific-popup.min.js"></script>
    <script src="js/bootstrap-datepicker.min.js"></script>
    <script src="js/swiper.min.js"></script>
    <script src="js/aos.js"></script>
  
    <script src="js/picturefill.min.js"></script>
    <script src="js/lightgallery-all.min.js"></script>
    <script src="js/jquery.mousewheel.min.js"></script>
  
    <script src="js/main.js"></script>
    
    <script>
      $(document).ready(function(){
        $('#lightgallery').lightGallery();
      });
    </script>
      
    </body>
  </html>
